besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -971,10 +972,16 @@ void setBannedNodeIds(final List<String> values) {
   private final Boolean migratePrivateDatabase = false;
 
   @Option(
-      names = {"--privacy-flexible-groups-enabled", "--privacy-onchain-groups-enabled"},
+      names = {"--privacy-flexible-groups-enabled"},
       description = "Enable flexible (onchain) privacy groups (default: ${DEFAULT-VALUE})")
   private final Boolean isFlexiblePrivacyGroupsEnabled = false;
 
+  @Option(
+      hidden = true,
+      names = {"--privacy-onchain-groups-enabled"},
+      description = "Enable flexible (onchain) privacy groups (default: ${DEFAULT-VALUE})")
```

Comment:
maybe add something in the description "this option has been replaced by --privacy-flexible-groups-enabled"

Reply:
50fc8f4